src/Employer.Web/Views/TrainingProvider/Confirm.cshtml

```diff
@@ -5,7 +5,21 @@
         <h1 class="heading-large">@Model.Title</h1>
         <h1 class="heading-large">Confirm training provider</h1>
 
+        <div class="verification-summary">
+            @*<h2 class="heading-medium error-summary-heading" id="verification-summary-heading-example-1">
+                Selected training provider
+            </h2>*@
+
+            <h3 class="heading-small">@Model.ProviderName</h3>
+            <h3 class="heading-small">@Model.ProviderAddress</h3>
+            <h3 class="heading-small">UKPRN: @Model.Ukprn</h3>
+        </div>
+
         <form asp-route="@RouteNames.TrainingProvider_Confirm_Post">
+            <input asp-for="ProviderName" type="hidden" />
+            <input asp-for="ProviderAddress" type="hidden" />
```

Comment:
I wouldn't rely on ProviderName and ProviderAddress in hidden fields.  Just get them again on the Post